ui/app/templates/jobs/job/services.hbs

```diff
@@ -0,0 +1,54 @@
+{{page-title "Job " this.model.name " services"}}
```

Comment:
In the interest of simplifying things, you could consider a template with aliases to your model properties here. See for example how `controllers/jobs/job/clients` handles this: https://github.com/hashicorp/nomad/blob/main/ui/app/controllers/jobs/job/clients.js#L64-L65

You would have `<JobSubnav @job={{this.job}} />` and a page title of `"Job" this.job.name` which is a little more clear to future coders.

But the biggest benefit, means that if we ever wanted to use this template to list the services of a job from outside of that job context (say, a "Services meta page" or something) we could make a change to this file's controller instead of remapping the route model.